node/clients/checkout.ts

```diff
@@ -80,17 +86,6 @@ export class Checkout extends JanusClient {
       { metric: 'checkout-setOrderFormCustomData' }
     )
 
-  public updateItems = (
-    orderFormId: string,
-    orderItems: Array<Omit<OrderFormItemInput, 'id'>>,
-    splitItem: boolean
-  ) =>
-    this.post<CheckoutOrderForm>(
-      this.routes.updateItems(orderFormId),
```

Comment:
What do you think about changing the route used here to the one used by `addItem`? So we wouldn't need to remove this method and, with that, we wouldn't lose this metric `checkout-updateItems`.

Reply:
And also we wouldn't need to have [this logic on LOC 63](https://github.com/vtex-apps/checkout-graphql/pull/125/files#diff-0a017c0453694f2c53c4774f6261676f4692532b233e1b747bbb77b6542e259cR63)